src/main/groovy/lang/ObjectRange.java

```diff
@@ -499,7 +512,7 @@ public List step(int step) {
      * @param value the value to increment
      * @return the incremented value
      */
-    protected Object increment(Object value) {
+    private Object increment(Object value) {
```

Comment:
not sure if you forgot a force push, but this commit should not change visibility. The groovy-team cherry-picks commits, so they must be valid individually